infra/periodic_killer.tf

```diff
@@ -8,16 +8,26 @@ resource "google_service_account" "periodic-killer" {
   account_id = "periodic-killer"
 }
 
+locals {
+  accounts_that_can_kill_machines = [
+    # should reference google_project_iam_custom_role.periodic-killer.id or
+    # something, but for whatever reason that's not exposed.
+    "serviceAccount:${google_service_account.periodic-killer.email}",
+
+    "user:gary.verhaegen@digitalasset.com",
+    "user:moritz.kiefer@digitalasset.com",
+  ]
+}
+
 resource "google_project_iam_member" "periodic-killer" {
-  # should reference google_project_iam_custom_role.periodic-killer.id or
-  # something, but for whatever reason that's not exposed.
-  role   = "projects/da-dev-gcp-daml-language/roles/killCiNodesEveryNight"
-  member = "serviceAccount:${google_service_account.periodic-killer.email}"
+  count  = "${length(local.accounts_that_can_kill_machines)}"
+  role   = "projects/da-dev-gcp-daml-language/roles/killCiNodes"
```

Comment:
Can we reference this through `${google_project_iam_custom_role.periodic-killer.id}`?

Reply:
Yes we can. Thanks! No clue why I did not do that at the time.